doc/fee-bumping-reserves.md

```diff
@@ -0,0 +1,43 @@
+# Fee-Bumping Reserves Recommendations for Anchor Outputs Channels
+
+The Lightning Dev Kit offers the ability to its users to open Lightning [anchor outputs](https://bitcoinops.org/en/topics/anchor-outputs/) channels (`option_anchors_zero_fee_htlc_tx`) with its counterparties.
+
+Under the Lightning security model, commitment and second-stage HTLC transactions (BOLT3's `HTLC-timeout` and `HTLC-preimage` and the corresponding spends on counterparty commitment transaction) must be realized
+in a time-sensitive manner if there are pending HTLCs. Lack of chain confirmation before HTLC expiration timelocks can constitute a loss of funds, as the counterparty can claim the HTLC output with a [competing
+transaction](https://bitcoinops.org/en/blog/waiting-for-confirmation/#policy-as-an-interface). Failure of confirmation might be provoked by blockspace demand spikes provoked by unrelated Bitcoin on-chain transaction
```

Comment:
I don’t know if you have read the link, I think the following excerpt is a nice reminder about Lightning security model about time-sensitive transaction. 

"Contracting protocols are even more intimately dependent on policies related to prioritization because enforceability on-chain depends on being able to get transactions confirmed quickly. In adversarial environments, cheating counterparties may have an interest in delaying a transaction’s confirmation, so we must also think about how quirks in the transaction relay policy interface can be used against a user.”

Do you think I should rather take a digest in `fee-bumping-reserves.md` rather than quoting a link ?

Reply:
Which is nice to refresh users memory on anchor outputs channels and the competing balance interest ?

Reply:
>  the fact that LDK itself also has to consider transaction standardness policies seems entirely unrelated?

I wish it was that simple, though e.g with nversin=3 there is a limit on the number of package descendant, and therefore how many commitment a CPFP can cover as one example coming out of mind.

Though I think you’re right in the let’s start simple approach, removed the whole section. I still added a warning at the end “Those recommendations should be also revised in function of the interactions with transaction standardness policy and its evolutions (e.g number of transactions allowed to be relayed as a package and as such how many LN commitment one CPFP can cover)”.